autoPyTorch/data/tabular_target_validator.py

```diff
@@ -88,10 +95,9 @@ def _fit(
             # We should not reach this if statement as we check for type of targets before
             raise ValueError("Multi-dimensional classification is not yet supported. "
                              "Encoding multidimensional data converts multiple columns "
-                             "to a 1 dimensional encoding. Data involved = {}/{}".format(
-                                 np.shape(y_train),
-                                 self.type_of_target
-                             ))
+                             "to a 1 dimensional encoding. Data involved = {}/{}".format(np.shape(y_train),
```

Comment:
Actually, we would like to shift to using `f-strings`. Could you make one here? You can also just undo this change and we can fix the f-strings as part of a later pull request.